labonneboite/importer/jobs/populate_flags.py

```diff
@@ -138,12 +145,19 @@ def dump():
 
 def make_link_file_to_new_archive(archive_path):
     link_path = os.path.join(settings.BACKUP_FOLDER, "latest_data.tar.bz2")
+
     try:
         os.remove(link_path)
     except OSError:
-        pass
-    # this is a hard link, not a symlink
-    os.link(archive_path, link_path)
+        pass  # link_path did not already exist
+    
+    try:
+        # this is a hard link, not a symlink
+        # hard linking fails on Vagrant, error seems to be known:
+        # https://github.com/pypa/setuptools/issues/516
+        os.link(archive_path, link_path)
```

Comment:
omg

Reply:
so what's your suggestion? let's get rid of hard link right? so 2000 ...

Reply:
i have no idea why legacy code used a hard link here btw